### PR TITLE
Some fixes for issue: https://github.com/telefonicaid/fiware-connectors/...

### PR DESCRIPTION
**Pull request by vgarciag**
...issues/313
Fix issue [#313](https://github.com/telefonicaid/fiware-connectors/issues/313)

### CODE REVIEW

File: flume/conf/agent.conf.template [hunk 1]
```diff
@@ -15,7 +15,7 @@
 # For those usages not covered by the GNU Affero General Public License please contact with iot_support at tid dot es
 
 #=============================================
-# To be put in APACHE_FLUME_HOME/conf/cygnus.conf
```
Comment (fgalan) [inline flume/conf/agent.conf.template]: The CHANGES_NEXT_RELEASE entry is missing, although maybe in this case is not needed if I understanding correctly and thi PR fix some problem introduce in a very recent PR.

Just a thought... mark this as NTC if it doesn't make sense to add entry to CHANGES_NEXT_RELEASE.

Comment (vgarciag) [inline flume/conf/agent.conf.template]: NTC

Yes, this PR solves some issues introduced in PR [#307](https://github.com/telefonicaid/fiware-connectors/pull/307)

**Comment by fgalan** [comment]
LGTM (not doing merge yet, until @frbattid can have a look)

**Comment by frbattid** [comment]
@vgarciag , did you tested it in a clean installation, free of previously installed configurations? If you did, LGTM :)

**Comment by vgarciag** [comment]
yes, I did it.

**Pull request merged**